### ISSUE TITLE
Reproducibility issue

### ISSUE THREAD

**Issue by juliohm** [JuliaDynamics/DrWatson.jl#294]
**Describe the bug**
Even if we follow all directions in the docs, the project is not fully reproducible.

**Minimal Working Example**
Add packages to the global environment @1.6 in dev version. Likewise add the same packages to the DrWatson project environment. After activation with `@quickactivate` Julia will still try to use the version in the global environment.

I think this issue is related to the load path for packages, it is stacked as far as I remember: https://youtu.be/q-LV4zoxc-E

Perhaps `@quickactivate` should do some additional magic to clean up the load path before calling activate? Or a new command in DrWatson.jl to clean up the path and make it empty?

**Comment by juliohm** [comment]
Yes I guess the problem is Pluto specific but we are not using their
package manager. Anytime we call @quickactivste Pluto recognizes that we
are handling the environment manually and switches to vanilla Julia package
management.

**Comment by Datseris** [comment]
Inside Pluto, after quickactivate, what does `projectname()` return?

**Comment by juliohm** [comment]
This LOAD_PATH explained in 8:12s: https://youtu.be/q-LV4zoxc-E?t=492

**Comment by fonsp** [comment]
Can you share the notebook file, your julia version and the contents of `.julia/environments`? Which package is not being loaded with the correct version? How can I tell?
